android/app/src/main/kotlin/coop/polypoly/polypod/polyNav/PolyNav.kt

```diff
@@ -71,7 +71,12 @@ open class PolyNav(
                 throw Error("File copy error")
             }
             val newId = UUID.randomUUID().toString()
-            ZipTools.unzipAndEncrypt(inputStream, context, newId)
+            var targetPath = newId
+            val featureName = Preferences.currentFeatureName
+            if (featureName != "") {
```

Comment:
Using an empty string as marker of unexistence, how idiomatic is it? Mostly referring to lin 82 above, though.

Reply:
It does seem a bit weird - it looks like this is all made to work even if the feature name cannot be determined. I suppose `""` might actually technically work, but I'd vote for something a little more clear, like `unknownFeature`. That way, features whose name cannot be read (for whatever reason) can still access the file system to some degree. Suppose I'd prefer an error though.